src/html/popup.html

```diff
@@ -1,33 +1,69 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
   <head></head>
   <body>
     <div class="uk-container">
-      <div class="uk-margin uk-margin-top">
-        <textarea id="js-title" class="uk-input" type="text" placeholder="title"></textarea>
+      <div class="uk-margin-top">
+        <button
+          id="js-save"
+          class="uk-icon-button uk-icon-large uk-align-center uk-margin-small"
+          uk-icon="bookmark"
+        />
       </div>
-      <div class="uk-margin">
-        <textarea id="js-abst"
-          class="uk-textarea"
-          rows="7"
-          placeholder="abstract"
-        ></textarea>
+      <div class="uk-margin-bottom">
+        <label class="uk-form-label" for="form-stacked-text">database</label>
+        <select class="uk-select" id="js-select-database"></select>
       </div>
       <div class="uk-margin">
-        <select class="uk-select" id="js-select-database"></select>
-        <div id="js-chip-container"></div>
         <progress
           class="uk-progress uk-width-1-1 uk-margin"
           id="js-progressbar"
           value="0"
           max="100"
         ></progress>
-        <button id="js-save" class="uk-icon-button uk-icon-large uk-align-center uk-margin-bottom" uk-icon="heart" />
+      </div>
+      <div class="uk-margin">
+        <label class="uk-form-label" for="form-stacked-text">title</label>
+        <textarea
+          id="js-title"
+          class="uk-input"
+          type="text"
+          placeholder="title"
+        ></textarea>
+      </div>
+      <div class="uk-margin">
+        <label class="uk-form-label" for="form-stacked-text">abstract</label>
+        <textarea
+          id="js-abst"
+          class="uk-textarea"
+          rows="4"
+          placeholder="abstract"
+        ></textarea>
+      </div>
+      <div class="uk-margin">
+        <label class="uk-form-label" for="form-stacked-text">comment</label>
+        <textarea
+          id="js-comment"
+          class="uk-input"
+          type="text"
+          placeholder="comment"
+        ></textarea>
+      </div>
+      <!-- <div class="uk-margin">                                                       -->
+      <!--   <label class="uk-form-label" for="form-stacked-text">published date</label> -->
+      <!--   <input                                                                      -->
+      <!--     id="js-published"                                                         -->
+      <!--     class="uk-input"                                                          -->
+      <!--     type="text"                                                               -->
+      <!--     placeholder="published"                                                   -->
+      <!--   ></input>                                                                   -->
+      <!-- </div>                                                                        -->
```

Comment:
I decided not to display "published" field in the user interface because:
1) this field should not be edted by hand.
2) maximum height of the popup window is limited (600px)